README.md

```diff
@@ -2,182 +2,62 @@
-custom_edit_url: https://github.com/facebook/docusaurus/edit/main/docs/api-doc-markdown.md
-description: How do I find you when I cannot solve this problem
-keywords:
-  - docs
-  - docusaurus
-image: https://i.imgur.com/mErPwqL.png
-slug: /myDoc
-last_update:
-  date: 1/1/2000
-  author: custom author name
----
-
-# Markdown Features
-
-My Document Markdown content
-```
-
-### Descriptions
-
-- Metadata descriptions are super important in making our documentation easy to find using search engines. To maximize our SEO, _keep the descriptions between [140 and 160 characters](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwigg6Og56brAhUNi1wKHULsAHEQFjAGegQIDBAG&url=https%3A%2F%2Fmoz.com%2Flearn%2Fseo%2Fmeta-description&usg=AOvVaw3L26bXhHZTd0wYDM_5xtJ9) whenever possible_.
-
-    Of course, when there just isn't enough to say, don't waffle - it's not a university essay.
-
-    GOOD:
-
-    > "Store anything from images and key-value pairs to structured output data. Learn how to access and manage your stored data from the Apify platform or via API."
-
-    AVOID:
-
-    > "Apify storage docs."
-
-- Avoid using the word "_documentation_".
-
-    GOOD:
-
-    > "Learn how to develop, run and share serverless cloud programs. Create your own web scraping and automation tools and publish them on the Apify platform."
-
-    AVOID:
-
-    > "Documentation of Apify Actors - the easy way to build serverless cloud programs."
-
-- Avoid _keyword stuffing_, i.e., repeating the article's name too much; view the [Wikipedia article](https://en.wikipedia.org/wiki/Keyword_stuffing) for more.
-
-    GOOD:
-
-    > Title: Proxy
-    >
-    > Description: Learn how to anonymously access websites when running web scraping or automation jobs. Prevent IP address-based blocking using IP address rotation.
-
-    AVOID:
-
-    > Title: Proxy
-    >
-    > Description: Learn how to use Apify Proxy. Prevent IP address-based blocking using a proxy. Apify Proxy helps you bypass security.
-
-- Phrase the descriptions in a way that answers a question that the person using the search engine might have.
-
-    GOOD:
-
-    > Learn how to make your Actor available to the public or keep it private. Prepare your Actor for Apify Store with a description and README file.
-
-    AVOID:
-
-    > Description of the processes regarding optimizing and preparing for publishing of one's Actor in Apify Store.
-
-### Screenshots
-
-Documentation has both light and dark themes. However, to keep the screenshots consistent and easily manageable, use the light theme for all screenshots.
-
-All screenshots should have a meaningful alt text to accommodate screen readers.
-
-If you add arrows or other indicators to the screenshots, they should be red for high contrast and visibility on the light theme.
+- For contribution questions, see our [Contributing Guidelines](CONTRIBUTING.md)
+- For technical issues, create a GitHub issue
```

Comment:
If I recall there's a direct link to create new issue in a repository

Reply:
added